### ISSUE TITLE
[LOC] Back-Translation "regsvr32.exe ボタン" → "regsvr32.exe wuaueng.dll"

### ISSUE THREAD

**Issue by hyoshioka0128** [MicrosoftDocs/windows-itpro-docs#3575]
https://docs.microsoft.com/ja-jp/windows/deployment/update/windows-update-resources

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: 886426b5-8548-1cfc-ccd2-fd03e549c967
* Version Independent ID: 599aea76-47e4-dccc-799c-2d8acf549546
* Content: [Windows Update - その他のリソース](https://docs.microsoft.com/ja-jp/windows/deployment/update/windows-update-resources)
* Content Source: [windows/deployment/update/windows-update-resources.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/deployment/update/windows-update-resources.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @kaushika-msft
* Microsoft Alias: **elizapo**

**Comment by e0i** [comment]
@officedocsbot assign @e0i
@officedocsbot assign @dongmicro
@officedocsbot add label localization

**Comment by e0i** [comment]
See: #3567

**Comment by e0i** [comment]
@officedocsbot assign @MollyMalone

**Comment by e0i** [comment]
@officedocsbot unassign @MollyMalone
@officedocsbot assign @TinaMcN

**Comment by e0i** [comment]
@officedocsbot unassign @dongmicro

**Comment by TinaMcN** [comment]
It seems to be an issue with the machine translation tool.  I logged a bug for the tool owners to investigate.

**Comment by illfated** [comment]
After the merge of my PR on 2019-09-05 (September 5, 2019), commit https://github.com/MicrosoftDocs/windows-itpro-docs/commit/00ff5b9219391031ac3240c7eb0f806ee38c952e, **this issue should be OK to close** now that the entire list of `regsvr32.exe` commands is wrapped in a code block like this:
```
regsvr32.exe atl.dll
regsvr32.exe urlmon.dll
regsvr32.exe mshtml.dll
regsvr32.exe shdocvw.dll
regsvr32.exe browseui.dll
regsvr32.exe jscript.dll
regsvr32.exe vbscript.dll
regsvr32.exe scrrun.dll
regsvr32.exe msxml.dll
regsvr32.exe msxml3.dll
regsvr32.exe msxml6.dll
regsvr32.exe actxprxy.dll
regsvr32.exe softpub.dll
regsvr32.exe wintrust.dll
regsvr32.exe dssenh.dll
regsvr32.exe rsaenh.dll
regsvr32.exe gpkcsp.dll
regsvr32.exe sccbase.dll
regsvr32.exe slbcsp.dll
regsvr32.exe cryptdlg.dll
regsvr32.exe oleaut32.dll
regsvr32.exe ole32.dll
regsvr32.exe shell32.dll
regsvr32.exe initpki.dll
regsvr32.exe wuapi.dll
regsvr32.exe wuaueng.dll
regsvr32.exe wuaueng1.dll
regsvr32.exe wucltui.dll
regsvr32.exe wups.dll
regsvr32.exe wups2.dll
regsvr32.exe wuweb.dll
regsvr32.exe qmgr.dll
regsvr32.exe qmgrprxy.dll
regsvr32.exe wucltux.dll
regsvr32.exe muweb.dll
regsvr32.exe wuwebv.dll
```
This list is now clearly visible as a code block, both in the originating /en-us/ page and the translated pages.

**Comment by hyoshioka0128** [comment]
@illfated  - Fixed.
Closed #3575
Thanks.
